### PR TITLE
Added option for display name

### LINKED ISSUE #41: Allow optional "Display Name"
Allow user to optionally alter how a share is named on the sidebar.

### PR DESCRIPTION
**Pull request by allenktv**
Feature for adding an optional Display Name to show up in files app.

Referencing #41, still need to revisit UX

### CODE REVIEW

File: app/samba.js [hunk 1]
```diff
@@ -233,8 +233,8 @@ SambaClient.prototype.mount_ = function(shareInfo, isNewMount) {
   // TODO(zentaro): Maybe support a friendly display name?
```
Comment (illiminable) [inline app/samba.js]: Can you remove this todo

File: app/window.html [hunk 2]
```diff
@@ -58,6 +58,9 @@
           <paper-input label="Share Path" id="sharePath" spellcheck="false" maxlength="512" autofocus></paper-input>
           <paper-tooltip for="sharePath" position="top" offset="0">The share path can either be in the Windows(tm) format eg. \\server\share or an smb URL eg. smb://server/share</paper-tooltip>
         </div>
+        <div>
+          <paper-input label="Display Name (Optional)" id="displayName" spellcheck="false" maxlength="512"></paper-input>
```
Comment (allenktv) [inline app/window.html]: Gotcha, will do. thanks!

**Comment by allenktv** [comment]
I updated the PR with the comments, will start looking into the suggested UX for display name field

**Comment by allenktv** [comment]
Added UX changes, let me know if what I did works!

**Pull request merged**